Next Migration tool (#126)
stoplightio/platform-docs: 6.-migrations/next.md

Now you'll need to retrieve your NEXT access token to start the migration process. In a new browser tab, navigate to your [account settings in NEXT](https://next.stoplight.io/profile/access-tokens) and create an API token.

![](../assets/images/next-account-settings.png)

![](../assets/images/next-access-token.png)

Copy the first access token and paste it into the input on the NEXT Migration screen in Stoplight Platform. If successful, you'll see a list of all the NEXT orgs you own and all of their projects.

Choose the NEXT projects you want to migrate to your Stoplight Platform workspace. After selecting at least one project, you'll see a button to migrate them.

![NEXT migration button](../assets/images/next-migration-button.png)

Wait a few moments for the projects to migrate and you'll see them appear in your left sidebar. Success!

You can continue this migration process as many times as you want. If you migrate the same project multiple times, **it will overwrite the existing project data in Stoplight Platform**. This will allow you to migrate projects at your own pace until NEXT is officially shutdown.

## Limitations

### Migration tool

> The NEXT Migration tool can migrate all of your NEXT projects, including all versions, OpenAPI files, Markdown files, and the most recently published Hub file.

Reviewer: Is it always doing most recently published now?

Author: ```suggestion
The NEXT Migration tool can migrate all of your NEXT projects, including all versions, OpenAPI files, Markdown files, and a single Hub file per project.
```

Author: No, but I explain it better down below on L56: https://github.com/stoplightio/platform-docs/pull/126/files/17f14a404db4f6b09a4f26ab8a322ccc5db6c3b2#diff-b151296f24abca87088b2331db4051903503e4939d49b0b7d1f6b1d3eb7162b0R56

However, there are a some items that won't be migrated:

- **Members** - the roles and permission system is slightly different in Stoplight Platform (see FAQ below), so Stoplight won't be able to migrate your teammates. You can [configure an allowed email domain](../2.-workspaces/allowed-email-domains.md) so your teammates can automatically join your workspace or [invite them manually by email](../2.-workspaces/d.inviting-your-team.md).

> - **Teams** - Stoplight is currently developing [Groups](https://stoplightio.productboard.com/portal/4-stoplight-platform/tabs/b52a696d-dde6-48ca-a1b4-deacf9610b4e/features/5739494/portal/expanded) functionality.

Reviewer: `https://roadmap.stoplight.io/c/137-workspace-groups`

Author: ```suggestion
- **Teams** - Stoplight is currently developing [Groups](https://roadmap.stoplight.io/c/137-workspace-groups) functionality.
```

Reviewer: 🤦

- **Hosted docs** - Stoplight won't be able to migrate your hosted documentation settings, such as themes, [custom domains](../2.-workspaces/j.custom-domains.md), [authentication](../2.-workspaces/e.configuring-authentication.md), and [analytics](../4.-documentation/e.configure-analytics.md). You will need to reconfigure these in your new Workspace's settings.
- **HTML, CSS and JS** - Stoplight Platform does not support HTML, CSS or JS. If you need a fully customizable documentation, please take a look at our open source tool, [Elements](https://stoplight.io/open-source/elements/) to embed your API reference documentation into a custom developer portal.
- **Scenarios** - Stoplight Platform does not support Scenarios. If you need contract testing, please take a look at our open source tool, [Prism](https://stoplight.io/open-source/prism/), and its built in [Validation Proxy](https://meta.stoplight.io/docs/prism/ZG9jOjk3-validation-proxy).

### Stoplight Platform vs Stoplight NEXT

> While the new Stoplight Platform includes a lot of new functionality, there are some features from NEXT that have not yet made their way over. We are taking this opportunity to re-imagine and improve the functionality listed below, and will be rolling it out through the rest of the year. If any of the features below are critical to your workflow, we suggest that you wait to migrate until the relevant features are released. You can continue to use Stoplight NEXT without disruption until then.

Reviewer: "If any of the features below are critical to your workflow, we suggest that you wait to migrate until the relevant features are released. You can continue to use Stoplight NEXT without disruption until then." <- suggest just removing this line at this point

Author: ```suggestion
While the new Stoplight Platform includes a lot of new functionality, there are some features from NEXT that have not yet made their way over. We are taking this opportunity to re-imagine and improve the functionality listed below, and will be rolling it out through the rest of the year.
```

| [~~Docs: Table of Contents ordering~~](https://roadmap.stoplight.io/c/59-customize-docs-sidebar-ordering)          |  Launched   | Completely custom ordering of your docs sidebar. By default, files will be organized alphabetically, matching the order in the filesystem |
| [~~Design: Shared parameters/responses~~](https://roadmap.stoplight.io/c/138-support-for-openapi-shared-responses) |  Launched   | Form based editing for OpenAPI shared components                                                                                          |
| [~~Docs: Integrations~~](https://roadmap.stoplight.io/c/64-analytics-integrations)                                 |  Launched   | First class support for integrations like Google Analytics, Segment, Intercom, etc                                                        |
| [~~Docs: Custom CSS, JS, and HTML~~](https://roadmap.stoplight.io/c/57-embeddable-component-library)               |  Launched   | Elements: a rich component library for creating custom API docs site                                                                      |

> | [Docs: Try It OAuth token generation](https://roadmap.stoplight.io/c/58-request-maker-authentication)              | Short-term  | Generate OAuth 1 & 2 tokens for HTTP requests in Try It                                                                                   |

Reviewer: Should maybe update this to `considering`?

Author: ```suggestion
| [Docs: Try It OAuth token generation](https://roadmap.stoplight.io/c/58-request-maker-authentication)              | considering  | Generate OAuth 1 & 2 tokens for HTTP requests in Try It                                                                                   |
```
